meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-support/ti-mfgtool-files/ti-mfgtool-files_0.1.bb

```diff
@@ -0,0 +1,48 @@
+SUMMARY = "MFGTOOL Support Files and Binaries"
+LICENSE = "BSD-3-Clause & LGPLv2.1+"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/BSD-3-Clause;md5=550794465ba0ec5312d6919e203a55f9 \
+                    file://${COMMON_LICENSE_DIR}/LGPL-2.1-or-later;md5=2a4f4fd2128ea2f65047ee63fbca9f68"
+
+inherit deploy nopackages
+
+# Use installer initramfs
+INITRAMFS_IMAGE ?= "initramfs-ostree-lmp-recovery"
```

Comment:
It requires only the u-boot images, so this dependency shouldn't be initrd specific.

Reply:
fixed